File: Python/0233.py
busHeight = 437
crashNumber = 0

nBridges = int(input())
heights = list(map(int, input().split()))

for num, height in enumerate(heights, 1):
    if height <= busHeight:
        crashNumber = num
        break

if crashNumber == 0:
    print('No crash')
else:
    print('Crash', crashNumber)

File: C++/0233.cpp
#include <iostream>
using namespace std;

int main()
{
    int busHeight = 437, n, bridgeHeight;

    cin >> n;
    for (int i = 1; i <= n; ++i)
    {
        cin >> bridgeHeight;
        if (bridgeHeight <= busHeight)
        {
            cout << "Crash " << i << endl;
            return 0;
        }
    }
    cout << "No crash" << endl;

    return 0;
}
